conf.d/gitnow.fish

```diff
@@ -239,6 +239,15 @@ function upstream -d "Gitnow: Commit all changes and push them to remote server"
     push
 end
 
+function branch -d "GitNow: Creates a new Gitflow branch from current branch" -a xbranch
+    if not __gitnow_is_git_repository
+        __gitnow_msg_not_valid_repository "branch"
+        return
+    end
+
+    __gitnow_check_create_branch "$xbranch"
+end
+
```

Comment:
The `branch` command should have a completion, maybe similar to this one.

https://github.com/joseluisq/gitnow/blob/7e4d74603a70d04db8809444e837a49070aef0d4/completions/merge.fish#L7-L9